pkg/config/templates/harvester-config.yaml

```diff
@@ -9,7 +9,8 @@ spec:
       config:
         vip_interface: "{{ .MgmtInterface }}"
     service:
-      harvester:
-        vipMode: "{{ .VipMode }}"
-        vip: "{{ .Vip }}"
-        vipHwAddr: "{{ .VipHwAddr }}"
+      vip:
+        enabled: true
+        mode: "{{ .VipMode }}"
```

Comment:
Do we need to convert `dhcp` to `DHCP` here?

Reply:
We use lower case in the harvester chart.
https://github.com/harvester/harvester/pull/1188

Reply:
Got it, thanks!